src/AltinnCore/Designer/appsettings.json

```diff
@@ -32,6 +32,20 @@
   "ApplicationInsights": {
     "InstrumentationKey": "b1020135-1b69-4e4d-8b8e-217072c70879"
   },
+  "Integrations": {
+    "AzureCosmosDbSettings": {
+      "EndpointUri": "https://app-release-deployment.documents.azure.com:443/",
+      "MasterKey": "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```

Comment:
Is this not a secret? "Master keys provide access to all the administrative resources for the database account". I see the endprinturi are point to the actual azure db. Or is this a key to the local emulator?

Reply:
I think this is the master key for the cosmos db when running the emulator.

Reply:
@jezpoz is correct, this is the same master key for everyone who uses the emulator. That particular key is not a secret